docs/references/web-ui-server-env-vars.md

```diff
@@ -0,0 +1,89 @@
+---
+id: web-ui-server-env-vars
+title: Docker environmental variables for Web UI
+sidebar_label: Docker environmental variables
+description: How to set Docker environmental variables for Temporal Web UI.
+tags:
+  - docker
+  - webui
+---
+
+Docker containers can be configured for use in a production setting.
+Use `docker run` to configure the Web UI environmental variables.
+
+```
+docker run \
+    -e TEMPORAL_ADDRESS=127.0.0.1:7233 \
+    -e TEMPORAL_UI_PORT=8080 \
+    -e TEMPORAL_AUTH_ENABLED=true \
+    -e TEMPORAL_AUTH_PROVIDER_URL=https://accounts.google.com \
+    -e TEMPORAL_AUTH_CLIENT_ID=xxxxx-xxxx.apps.googleusercontent.com \
+    -e TEMPORAL_AUTH_CLIENT_SECRET=xxxxxxxxxxxxxxx \
+    -e TEMPORAL_AUTH_CALLBACK_URL=https://xxxx.com:8080/auth/sso/callback \
+    -e TEMPORAL_UI_ENABLED=true \
+    -e TEMPORAL_OPENAPI_ENABLED=true \
+    -e TEMPORAL_TLS_CA=../ca.cert \
+    -e TEMPORAL_TLS_CERT=../cluster.pem \
+    -e TEMPORAL_TLS_KEY=../cluster.key \
+    -e TEMPORAL_TLS_ENABLE_HOST_VERIFICATION=true \
+    -e TEMPORAL_TLS_SERVER_NAME=tls-server \
+    temporalio/ui:<tag>
+```
+
+The environmental variables are defined as follows:
+
+## `TEMPORAL_ADDRESS`
```

Comment:
Can we provide or link to places where there are some example values for any of these?

Reply:
I have the example above. I can poke around for other things